docs/generators/changelog/versions/2.13.md

```diff
@@ -26,12 +26,29 @@ TODO
 #### Meteor Version Release
 
 
+
+* `ddp-server@get-version`:
+    - Updated livedata server test to be more easily debbuged.
+
 * `Command line`:
     - Updated metatags for skeletons.
```

Comment:
Is this for adding the mobile meta tags? It might be nice to specify that.

Reply:
sure!